### PR TITLE
Add helper function `print_debug` for debug purposes

### PR DESCRIPTION
**Pull request by yabeow**
Hi there,

I've added an helper function called `print_debug` in DSL. This function will help to print DSL variables / outputs to stdout using `fmt.Println` function for debug purposes.

Note: I've tried to use your `gologger` package but it always requires a format string so I used the standard `fmt` package instead.

### CODE REVIEW

File: v2/pkg/operators/common/dsl/dsl.go [hunk 1]
```diff
@@ -245,6 +245,11 @@ var functions = map[string]govaluate.ExpressionFunction{
 		data := deserialization.GenerateJavaGadget(gadget, cmd, encoding)
 		return data, nil
 	},
+	// for debug purposes
+	"print_stdout": func(args ...interface{}) (interface{}, error) {
+		fmt.Println(args)
```
Comment (Ice3man543) [inline v2/pkg/operators/common/dsl/dsl.go:250]: We should add a message before printing the debug values, stating they are from print_debug function. Something like this @yabeow 

```go
	gologger.Info().Msgf("print_stdout debug value: ")
        fmt.Println(args)
```

File: v2/pkg/operators/common/dsl/dsl.go [hunk 2]
```diff
@@ -245,6 +245,11 @@ var functions = map[string]govaluate.ExpressionFunction{
 		data := deserialization.GenerateJavaGadget(gadget, cmd, encoding)
 		return data, nil
 	},
+	// for debug purposes
+	"print_stdout": func(args ...interface{}) (interface{}, error) {
```
Comment (forgedhallpass) [inline v2/pkg/operators/common/dsl/dsl.go:249]: if it's for debugging purposes, why not name it "debug" instead?

File: v2/pkg/operators/common/dsl/dsl.go (hunk 1 above)
Comment (yabeow) [reply v2/pkg/operators/common/dsl/dsl.go]: Nice suggestion! I've added this, please check again 👍

File: v2/pkg/operators/common/dsl/dsl.go (hunk 2 above)
Comment (yabeow) [reply v2/pkg/operators/common/dsl/dsl.go]: I've changed it into `print_debug` 👍  @forgedhallpass

**Comment by forgedhallpass** [comment]
lgtm!

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**